Jenkinsfile

```diff
@@ -1170,5 +1170,8 @@ def defaultCucumberFilterTags(env) {
   }
 
   // For all other branch builds, only run the @smoke tests by default
-  return '@smoke'
+  // return '@smoke'
+
+  // Temporarily run all tests on all branches
```

Comment:
Can we add to this comment what the criteria is for unwinding the temporary change?

In the unlikely case this gets missed or forgotten, it would be good for this to be self-describing on when "temporarily" ends.

Reply:
Updated the comment - lmk if that makes sense